sdcm/sct_events/database.py

```diff
@@ -308,6 +310,12 @@ def _end_event(event_type: Type[ScyllaDatabaseContinuousEvent], match: Match):
         if shard is not None:
             event_filter.filter_by_shard(shard)
 
+        if alert_name is not None:
+            event_filter.filter_by_alert(alert_name)
+
+        if starts_at is not None:
+            event_filter.filter_by_alert(starts_at)
```

Comment:
Shouldn't this be `event_filter.filter_by_start_at(start_at)`?

Reply:
ups. you are right. Fixed